docs-source/content/faq/resource-settings.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Pod Memory and CPU Resources"
+date: 2020-06-30T08:55:00-05:00
+draft: false
+weight: 25
+---
+
+### Contents
+
+ - [Introduction](#introduction)
+ - [Setting resource requests and limits in a domain resource](#setting-resource-requests-and-limits-in-a-domain-resource)
+ - [Determining pod Quality Of Service](#determining-pod-quality-of-service)
+ - [Java heap size and memory resource considerations](#java-heap-size-and-memory-resource-considerations)
+   - [Importance of setting heap size and memory resources](#importance-of-setting-heap-size-and-memory-resources)
+   - [Default heap sizes](#default-heap-sizes)
+   - [Configuring heap size](#configuring-heap-size)
+ - [CPU resource considerations](#cpu-resource-considerations)
+ - [Operator sample heap and resource configuration](#operator-sample-heap-and-resource-configuration)
+ - [Configuring CPU affinity](#configuring-cpu-affinity)
+ - [Measuring JVM heap, pod CPU, and pod memory](#measuring-jvm-heap-pod-cpu-and-pod-memory)
+ - [References](#references)
+
+### Introduction
+
+An operator creates a pod for each WebLogic Server instance and each pod will have a container. You can tune pod container memory and/or CPU usage by configuring Kubernetes resource requests and limits, and you can tune a WebLogic JVM heap usage using the `USER_MEM_ARGS` environment variable in your domain resource. A resource request sets the minimum amount of a resource that a container requires. A resource limit is the maximum amount of resource a container is given and prevents a container from using more than its share of a resource. Additionally, resource requests and limits determine a pod's Quality of Service.
```

Comment:
"An operator" -> "The operator"
I think I would say it like this, "The operator creates a container in its own Pod for each WebLogic Server instance."
"pod container memory" -> "container memory"
"and/or" -> "and"
"maximum amount of resource" -> "maximum amount of a resource"
I don't think "Quality of Service" should be capitalized here.
Also, we've been matching the style in the Kubernetes doc (on their site) and capitalized the names of all resources when specifically referenced as a resource, so all "pod" -> "Pod" and "domain resource" -> "Domain resource".

Reply:
I have made changes based on above review comments. Thanks.